ffac-wg-registration/files/lib/gluon/wg-registration/registration.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/sh
 
-if [ "$(uci get gluon.mesh_vpn.enabled)" == "true" ] || [ "$(uci get gluon.mesh_vpn.enabled)" == "1" ]; then
+if [ "$(uci get gluon.mesh_vpn.enabled)" = "true" ] || [ "$(uci get gluon.mesh_vpn.enabled)" = "1" ]; then
         # check if registration has been done since last boot
         if [ ! -f /tmp/WG_REGISTRATION_SUCCESSFUL ]; then
 		# Push public key to broker, test for https and use if supported
-		wget -q https://[::1]
+		wget -q "https://[::1]"
```

Comment:
Shouldn't this also be `if ! wget -q "https://[::1]"; then` ?

Reply:
Absolutely, thanks for spotting this

Reply:
I think I found an issue in this code actually. I will open a separate PR to address those

Reply:
See https://github.com/freifunk-gluon/community-packages/pull/58 for related discussion